sensor_msgs/PointCloud2 point_cloud
---
vision_msgs/BoundingBox3DArray detected_objects
